### PR TITLE
Just now added corresponding registry key  and  download link for tem…

### PR DESCRIPTION
**Pull request by RAJU2529**
…plates

I taken example from windows 10  v1903, i found  the corresponding registry keys  for   hide all  notifications  and  hide  non-critical notifications

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender Security Center\Notifications]
"DisableNotifications"=dword:00000001

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender Security Center\Notifications]
"DisableEnhancedNotifications"=dword:00000001

Also  i added  the download link for latest administrative templates for Windows 10 v1809.

So please add these registry keys and download link as my contribution in this document.
Thanking you

### CODE REVIEW

**Comment by illfated** [comment]
Just a small note and advice regarding PR titles (for future use): skip references to when the PR was created (now, just now, right now, and so on) as well as skipping personal pronouns (I, me, myself). PR and issue titles should be short, concise and technical, not personal notes.

File: windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ This can only be done in Group Policy.
 >
 >You must have Windows 10, version 1709 or later. The ADMX/ADML template files for earlier versions of Windows do not include these Group Policy settings. 
 
+0. Download the latest Administrative templates for windows 10 v1809 from below Microsoft official site
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]: Although `0` is a valid list number in programming and mathematics, it is not a practical number in literature. If you want to add something to the beginning of the list, make sure that it starts with 1 and that the list further down follows order.

Comment (RAJU2529) [reply windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]: @illfated . Thanks sir . I noted Down on the paper. I take care of these words.

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md (hunk 1 above)
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]:
```suggestion
1. Download the latest [Administrative Templates (.admx) for Windows 10, v1809](https://www.microsoft.com/download/details.aspx?id=57576).
```

File: windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md [hunk 2]
```diff
@@ -56,6 +56,10 @@ This can only be done in Group Policy.
...
+     **https://www.microsoft.com/en-us/download/details.aspx?id=57576**
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]:
```suggestion
```

File: windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md [hunk 3]
```diff
@@ -56,6 +56,10 @@ This can only be done in Group Policy.
...
+
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]:
```suggestion
```

File: windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md [hunk 4]
```diff
@@ -74,6 +78,8 @@ You can hide all notifications that are sourced from the Windows Security app. T
 This can only be done in Group Policy.
 
 >[!IMPORTANT]
+
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]:
```suggestion
```

File: windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md [hunk 5]
```diff
@@ -74,6 +78,8 @@ You can hide all notifications that are sourced from the Windows Security app. T
...
+
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]:
```suggestion
```

File: windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md [hunk 6]
```diff
@@ -56,6 +56,10 @@ This can only be done in Group Policy.
 >
 >You must have Windows 10, version 1709 or later. The ADMX/ADML template files for earlier versions of Windows do not include these Group Policy settings. 
 
+0. Download the latest Administrative templates for windows 10 v1809 from below Microsoft official site
+     **https://www.microsoft.com/en-us/download/details.aspx?id=57576**
+
+
 1.  On your Group Policy management machine, open the [Group Policy Management Console](https://technet.microsoft.com/library/cc731212.aspx), right-click the Group Policy Object you want to configure and click **Edit**.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]:
```suggestion
2.  On your Group Policy management machine, open the [Group Policy Management Console](https://technet.microsoft.com/library/cc731212.aspx), right-click the Group Policy Object you want to configure and click **Edit**.
```

Comment (nenonix) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]:
```suggestion
2.  On your Group Policy management machine, open the [Group Policy Management Console](https://technet.microsoft.com/library/cc731212.aspx), right-click the Group Policy Object you want to configure and click **Edit**.
```

Comment (nenonix) [reply windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]: The link in this step is outdated - maybe there is a better one?

File: windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md [hunk 7]
```diff
@@ -86,4 +92,14 @@ This can only be done in Group Policy.
 
 6.  Open the **Hide all notifications** setting and set it to **Enabled**. Click **OK**.
 
-7. [Deploy the updated GPO as you normally do](https://msdn.microsoft.com/library/ee663280(v=vs.85).aspx). 
+7.  Corresponding registry key  for **Hide all notifications**
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]:
```suggestion
7.  Use the following registry key and DWORD value to **Hide all notifications**.
```

File: windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md [hunk 8]
```diff
@@ -86,4 +92,14 @@ This can only be done in Group Policy.
...
+   
+    **[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender Security Center\Notifications]**
+    **"DisableNotifications"=dword:00000001**
+    
+8.   Corresponding registry key  for **Hide not-critical notifications**
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]:
```suggestion
8.  Use the following registry key and DWORD value to **Hide not-critical notifications**
```

**Comment by e0i** [comment]
@dansimp A community contributed PR that has passed copy review is available for technical review. Thanks.

**Comment by e0i** [comment]
@dansimp A community contributed PR that has passed copy review is available for technical review. Thanks.

**Comment by e0i** [comment]
@Dansimp Following up for updats.

**Comment by e0i** [comment]
@Dansimp Following up for updates.

**Pull request merged**